packages/expo/src/winter/TextDecoder.ts

```diff
@@ -0,0 +1,429 @@
+// A fork of text-encoding but with only UTF-8 decoder.
+// `TextEncoder` is in Hermes and we only need utf-8 decoder for React Server Components.
+//
+// https://github.com/inexorabletash/text-encoding/blob/3f330964c0e97e1ed344c2a3e963f4598610a7ad/lib/encoding.js#L1
+
+/**
+ * Checks if a number is within a specified range.
+ * @param {number} a The number to test.
+ * @param {number} min The minimum value in the range, inclusive.
+ * @param {number} max The maximum value in the range, inclusive.
+ * @returns {boolean} True if a is within the range.
```

Comment:
```suggestion
 * @param a The number to test.
 * @param min The minimum value in the range, inclusive.
 * @param max The maximum value in the range, inclusive.
 * @returns `true` if a passed number is within the specified range.
```

Code comments nits. We can avoid setting up the types, those are automatically resolved in docs from the TypeScript definitions.